{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -231,11 +232,11 @@ def test_permit(chain,token,owner):
     """
     spender = accounts[1]
     amount = 100
-    nonce = token.nonce(owner)
+    nonce = token.nonce
```

Comment:
Should this be `token.nonces(owner)` ?

Reply:
Yeah I had misunderstood this a bit, thanks! Updated in 89c3a1d